ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/OldApiUtils.java

```diff
@@ -112,6 +120,9 @@ static void handleOldApiResponse(final Server server, final RoutingContext routi
         response.end(responseBody);
       }
     }
+    pullQueryMetrics
+        .ifPresent(pullQueryExecutorMetrics -> pullQueryExecutorMetrics.recordResponseSize(
```

Comment:
Why not do recordLatency here too?  You can even record the start time here as well.  That way, they don't have to be sprinkled everywhere.  You would still have to handle the web socket since I don't believe it uses this path.

Reply:
Ideally we should measure every one of these endpoints in a generic fashion rather than having this special thing for pull queries, but maybe that is for a followup.